table-api/src/main/java/io/deephaven/api/updateby/UpdateByOperation.java

```diff
@@ -2101,6 +2100,224 @@ static UpdateByOperation RollingFormula(String timestampCol, long revTime, long
         return RollingFormulaSpec.ofTime(timestampCol, revTime, fwdTime, formula, paramToken).clause(pairs);
     }
 
+    // New methods for the multi-column formula
+
+    /**
+     * Create a {@link RollingFormulaSpec rolling forumla} using ticks as the windowing unit. Ticks are row counts and
+     * you may specify the previous window in number of rows to include. The current row is considered to belong to the
+     * reverse window, so calling this with {@code revTicks = 1} will simply return the current row. Specifying
+     * {@code revTicks = 10} will include the previous 9 rows to this one and this row for a total of 10 rows.
+     * <p>
+     * The provided {@code formula} should specify the output column name Some examples of formula are:
```

Comment:
Apply to all similar methods.
```suggestion
     * The provided {@code formula} should specify the output column name. Some examples of formula are:
```